remote.lua

```diff
@@ -33,28 +33,34 @@ end
 
 --
 -- Creates a sensor managed by a remote interface (another mod or script)
--- sensor_name: internal name of the sensor. Should be unique.
--- sensor_text: Text to display in the active gui
--- sensor_caption: Sensor setting name in the EvoGUI settings panel
--- sensor_color: Font color of the text to display in the active gui, optional, may be nil
--- example: remote.call("EvoGUI", "create_remote_sensor", "mymod_my_sensor_name", "Text: Lorem Ipsum", "[My Mod] Lorem Ipsum Text")
-local function create_remote_sensor(sensor_name, sensor_text, sensor_caption, sensor_color)
-    if not sensor_name then
-        evogui.log({"err_nosensorname"})
+-- sensor_data: a table with the following fields,
+--     mod_name: Name of the mod registering the sensor. Sensor will be removed if the mod is removed from the game.
+--     name: internal name of the sensor. Should be unique.
+--     text: Text to display in the active gui
+--     caption: Sensor setting name in the EvoGUI settings panel
+--     color: Font color of the text to display in the active gui, optional, may be nil
+--
+-- example: remote.call("EvoGUI", "create_remote_sensor", { "mod_name" = "my_mod",
+--                                                          "name" = "my_mod_my_sensor_name",
+--                                                          "text" = "Text: Lorem Ipsum",
+--                                                          "caption" = "[My Mod] Lorem Ipsum Text" })
```

Comment:
This documentation should reflect that the mod name gets added automatically, so the `caption` element can just be a plain display name (allowing localization): `"caption" = {"mymod_sensor_displayname"}`